geode-pulse/src/main/webapp/WEB-INF/spring-security.xml

```diff
@@ -20,15 +20,14 @@
 	xmlns:context="http://www.springframework.org/schema/context"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://www.springframework.org/schema/beans
-	http://www.springframework.org/schema/beans/spring-beans.xsd
+	http://www.springframework.org/schema/beans/spring-beans-4.3.xsd
```

Comment:
Sorry, didn't mean to commit this change.  I added it back in during development because Intellij don't property resolve the schema for me without the version.  Do you have this problem?  (Elements like `<headers>` are red in intellij without the version).